test/test_RTP.py

```diff
@@ -13,84 +13,177 @@
+                                        end_time=end_time, date_fmt=date_fmt,
+                                        channel=2, scale=scale,
+                                        cp_name=cp_name, color=color,
+                                        linestyle=linestyle,
+                                        linewidth=linewidth)
+
+    def test_parameters_time_series_channel1(self, parameters_scalar, gate,
+                                             scale, cp_name, color, linestyle,
+                                             linewidth, date_fmt, end_time,
+                                             start_time):
+        """
+
+        """
+        with warnings.catch_warnings(record=True):
+            pydarn.RTP.plot_time_series(data, parameter=parameters_scalar,
+                                        beam_num=15, gate=gate,
+                                        start_time=start_time,
+                                        end_time=end_time, date_fmt=date_fmt,
+                                        channel=1, scale=scale,
+                                        cp_name=cp_name, color=color,
+                                        linestyle=linestyle,
+                                        linewidth=linewidth)
+
+
+@pytest.mark.parametrize('fig_size', [(11, 10), (10, 11)])
+@pytest.mark.parametrize('watermark', [False, True])
+@pytest.mark.parametrize('boundary', [{'w_l': (0, 100),
+                                       'nave': (0, 30),
+                                       'elv': (0, 20),
+                                       'tfreq': (0, 15)}])
+@pytest.mark.parametrize('cmaps', [{'elv': plt.get_cmap('rainbow'),
+                                    'p_l': plt.get_cmap('rainbow'),
+                                    'vel': plt.get_cmap('rainbow')}])
+@pytest.mark.parametrize('lines', [{'nave': '--', 'tfreq': '.'}])
+@pytest.mark.parametrize('plot_elv', [False])
+@pytest.mark.parametrize('title', ['test test'])
+@pytest.mark.parametrize('background', ['grey'])
+@pytest.mark.parametrize('groundscatter', [False, 'grey'])
+@pytest.mark.parametrize('coords', [pydarn.Coords.RANGE_GATE,
+                                    pydarn.Coords.GROUND_SCATTER_MAPPED_RANGE])
+class TestSummaryPlots:
+
+    def test_params_summary_plots(self, fig_size, watermark, boundary, cmaps,
+                                  lines, plot_elv, title, background,
+                                  groundscatter, coords):
+        with warnings.catch_warnings(record=True):
+            pydarn.RTP.plot_summary(data, beam_num=15, channel=1,
+                                    figsize=fig_size, watermark=watermark,
+                                    boundary=boundary, cmaps=cmaps,
+                                    lines=lines, plot_elv=plot_elv,
+                                    title=title, background=background,
+                                    groundscatter=groundscatter, coords=coords)
```

Comment:
It keeps throwing up that there is an error on this line, but the error doesn't make sense to be coming from this line as its another color issue: 
`ValueError: '.' is not a valid value for color`
The only input that has a '.' is from 'lines'

Reply:
Looking further into this, I'm not sure but lines wants colour values, so if I replace '--' and '.' with 'r' and 'g' then it runs with no errors, I guess lines means the line colour and not the line style?